tests/update/patch_sle.pm

```diff
@@ -117,6 +117,12 @@ sub patching_sle {
     set_var("VIDEOMODE", $orig_videomode);
     set_var("SCC_REGISTER", $orig_scc_register);
 
+    # disable multiversion for kernel-default based on bsc#1097111, for migration continuous cases only
+    if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {
+        script_run('sed -i s/multiversion/#multiversion/g /etc/zypp/zypp.conf');
```

Comment:
We'd better to add a check for the multiversion is enabled or not, and then take action base on the check.

Besides, why not add a function to do this? So to avoid code duplicate.

Reply:
> We'd better to add a check for the multiversion is enabled or not, and then take action base on the check.
> 
> Besides, why not add a function to do this? So to avoid code duplicate.

Do you mean show the /etc/zypp/zypp.conf?
The function is very short, and there are two functions, add '#' and remove '#', I think no need to add two functions for it.

Reply:
You can update it to ''sed -i s/^multiversion/#multiversion/g /etc/zypp/zypp.conf'.

For the function:
Ex:

my $sed_para = get_var('PUBLISH_HDD_1') && check_var('VERSION', get_var('UPGRADE_TARGET_VERSION') ? 's/^#multiversion/multiversion/g' : 's/^multiversion/#multiversion/g';
script_run('sed -i $sed_para /etc/zypp/zypp.conf');

PUBLISH_HDD_1 used to ensure it is the 2nd step, check_var('VERSION', get_var('UPGRADE_TARGET_VERSION') used to judge it is before migration or after migrartion.